spec/support/spec_config.rb

```diff
@@ -363,15 +363,24 @@ def retry_writes_options
   # Base test options.
   def base_test_options
     {
-      max_pool_size: 1,
+      # Automatic encryption tests require a minimum of two connections,
+      # because the driver checks out a connection to build a command,
+      # and then may need to encrypt the command which could require a
+      # query to key vault connection triggered from libmongocrypt.
+      # In the worst case using FLE may end up doubling the number of
+      # connections that the driver uses at any one time.
```

Comment:
This should probably make its way into our documentation in some way.

Reply:
I have added a line to the [Client documentation](https://github.com/mongodb/mongo-ruby-driver/pull/1646/files#diff-9b873200a76992a56a5c5fa0fd7914b2R392). Let me know if you'd like to see this in other parts of the documentation as well.

Reply:
This could go into the tutorial but since we don't currently have an fle section in the tutorial let's come back to this later.